### PR TITLE
export history to csv

### LINKED ISSUE #2764: feat: export account history into csv
### Subject of the issue
Add an option to account history to export data into a csv file (e.g. with `react-csv`)

Design (see Figma link)

### Deliverables
- Add header/subtitle according to Figma design
- Add Export button according to design for both dark and light theme
- Implement functionality to export csv for selected table (Positions or Trades)

### What to export?
Same data as in table, plus:

- include transaction hash on export as a line item (currently hidden behind click on the trade itself)

### PR DESCRIPTION
**Pull request by pldespaigne**
fix #2764

### CODE REVIEW

File: packages/app/src/state/wallet/selectors.ts [hunk 1]
```diff
@@ -6,7 +6,8 @@ import type { RootState } from 'state/store'
 
 const SUPPORTED_NETWORKS = [1, 10, 5, 420]
 
-export const selectWallet = (state: RootState) => state.wallet.walletAddress ?? null
+export const selectWallet = (state: RootState) =>
+	'0xC2ecD777d06FFDF8B3179286BEabF52B67E9d991' ?? null
```
Comment (avclarke) [inline packages/app/src/state/wallet/selectors.ts:10]: Need to revert this

Comment (platschi) [inline packages/app/src/state/wallet/selectors.ts:10]: We need to revert this change for prod

**Review by platschi — CHANGES_REQUESTED** [review]

File: packages/app/src/sections/futures/TraderHistory.tsx [hunk 2]
```diff
@@ -73,7 +75,23 @@ const TraderHistory: FC<TraderHistoryProps> = memo(
...
+			result.forEach(
+				(row: (typeof result)[number]) =>
+					(csvData += `${new Date(row.timestamp).toISOString()},${row.marketShortName},${
+						row.status
+					},${row.trades},${row.totalVolume.toNumber()},${row.pnl.toNumber()},${row.pnlPct.slice(
+						1,
+						-2
+					)},${row.funding.toNumber()},${row.transactionHash}\n`)
+			)
+			dispatch(setCsvExportData(csvData))
```
Comment (avclarke) [inline packages/app/src/sections/futures/TraderHistory.tsx:91]: It would be nice if we could move all of this data formatting logic out of the component and into redux. So I think we could create a selector for the table data `selectPositionHistoryTableData` which we move the original table data formatting to and then `selectCsvExport` can just pull in the data from `selectPositionHistoryTableData` meaning we can also remove the `csvExport` field it's reducer.

**Review by avclarke — CHANGES_REQUESTED** [review]
Looks good but with the added formatting introduced for the csv export I think it's a good time to move all this data formatting over to a redux selector and out of the component. This will also remove the need to add the `csvExport` field, we can just use a selector.

**Review by avclarke — COMMENTED** [review]
Changes looks great, just a couple of clean up suggestions.

**Review by avclarke — APPROVED** [review]
Changes look good, nice work!

File: packages/app/src/sections/futures/TraderHistory.tsx [hunk 3]
```diff
@@ -1,7 +1,5 @@
-import { ZERO_WEI } from '@kwenta/sdk/constants'
-import { FuturesPositionHistory } from '@kwenta/sdk/dist/types'
-import { getMarketName, MarketKeyByAsset } from '@kwenta/sdk/utils'
-import { wei, WeiSource } from '@synthetixio/wei'
+import { FuturesMarketKey, FuturesPositionHistory } from '@kwenta/sdk/dist/types'
```
Comment (avclarke) [inline packages/app/src/sections/futures/TraderHistory.tsx:1]:
```suggestion
import { FuturesMarketKey, FuturesPositionHistory } from '@kwenta/sdk/types'
```

File: packages/app/src/utils/futures.ts [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import { ZERO_WEI } from '@kwenta/sdk/dist/constants'
```
Comment (avclarke) [inline packages/app/src/utils/futures.ts:1]:
```suggestion
import { ZERO_WEI } from '@kwenta/sdk/constants'
```

**Review by avclarke — CHANGES_REQUESTED** [review]
Just noticed there are a couple of places using `/dist` imports which cause build failures.

**Review by platschi — APPROVED** [review]
